### ISSUE TITLE
Dockerfile is not working

### ISSUE THREAD

**Issue by ak4zh** [fboulnois/pg_uuidv7#19]
Built the image using provided Dockerfile.

```bash
postgres=# CREATE EXTENSION pg_uuidv7;
ERROR:  extension "pg_uuidv7" is not available
DETAIL:  Could not open extension control file "/usr/share/postgresql/16/extension/pg_uuidv7.control": No such file or directory.
HINT:  The extension must first be installed on the system where PostgreSQL is running.
postgres=# SELECT uuid_generate_v7();
ERROR:  function uuid_generate_v7() does not exist
LINE 1: SELECT uuid_generate_v7();
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
```

**Comment by fboulnois** [comment]
See https://github.com/fboulnois/pg_uuidv7/issues/17 for temporary solutions. This will be fixed in the next version of the extension.

**Comment by fboulnois** [comment]
This issue is resolved as of v1.4.0.
